src/oslinfo/oslinfo.cpp

```diff
@@ -235,6 +244,7 @@ main (int argc, char *argv[])
                 "-h", &help, "Print help message",
                 "--help", &help, "",
                 "-v", &verbose, "Verbose",
+                "--bench", &benchmark, "Benchmark shader loading time",
```

Comment:
If you want to time what happens in ShadingSystem, then time in ShadingSystem, not OSLQuery. I think you could already accomplish this with

    testshade --stats myshader

and then the stats will already break out the time to read the shader master.

Apologies -- this is the one spot where I used "--stats". We should change that to "--runstats" for consistency. In contrast, oiiotool has both, because it's a separate operation to want statistics about the input images (--stats) versus time/memory/etc/ statistics about the running of the program itself (--runstats). We'll do a pass to make that consistent.

And I like --runstats better than --bench for two reasons: (1) it can incorporate things other than times; and (2) a proper benchmark would probably run lots of times and report the median or fastest time or whatever, whereas runstats just means tell me what happened on this one ordinary run.